src/werkzeug/debug/shared/debugger.js

```diff
@@ -1,3 +1,25 @@
+function fadeOut(el){
```

Comment:
```suggestion
function fadeOut(el) {
```
Can we use a more descriptive name than `el`?